.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
 install:
     - pip install pipenv
     - pipenv install --dev
+    - pipenv graph
```

Comment:
Can you remove this due to #32? Would conflict anyways when rebasing.

Reply:
Done.